### DISCUSSION TITLE
hnsw index not supported error in Azure Cosmos DB for MongoDB (vCore)

### DISCUSSION THREAD

**Discussion by Divakar-kumar** [microsoft/AI-Chat-App-Hack#59] [Q&A] [answered]
Hi,

I have enabled **HNSW index for vector search** preview feature under my subscription and tried to create index of kind **'hnsw'**, but received this error "Command createIndexes failed: hnsw index is not supported yet."

I used MongoDb.Driver 2.23.1 version. I would need assistance on this to proceed further.

**[Update] :  For now, I switched to vector-ivf mode**

**Comment by pamelafox** [comment]
cc @khelanmodi to look into the issue.

I'm glad you got IVF working! That should work well.

**Comment by khelanmodi** [comment, marked as answer]
Hey @Divakar-kumar, currently, HNSW is only available for M50 and higher cluster tiers. Please upgrade your tier using the Scale blade  in the portal.

**Reply by Divakar-kumar** [reply]
Hi @khelanmodi , Sure thanks for letting me know.
